app/src/store/actionCreators.test.js

```diff
@@ -0,0 +1,68 @@
+import { initialState } from './initialState';
+import { getMovies, deleteMovie, addMovie, updateMovie } from './actionCreators';
+import configureStore from 'redux-mock-store';
+import fetchMock from 'fetch-mock';
+import thunk from 'redux-thunk';
+import * as apiMap from '@store/apiMap';
+import { Movie } from '@models';
+
+describe('movies actions', () => {
+    let store;
+    beforeEach(() => {
+        store = configureStore([thunk])(initialState);
+    })
+
+    apiMap.getMoviesUrl =  jest.fn(() => 'url');
+    apiMap.deleteMovieUrl =  jest.fn(id => 'delete_url/' + id);
+    apiMap.addMovieUrl =  jest.fn(() => 'add_url');
+    apiMap.updateMovieUrl =  jest.fn(() => 'update_url');
+
+    fetchMock.get('url', {
+        body: { data: ['movie1', 'movie2'] },
+        headers: { 'content-type': 'application/json' }
+    })
+
+    let expectedAction = { 
+        type: 'SET_MOVIES', 
+        payload: [ 'movie1', 'movie2' ] 
+    };
+
+    it('get movies', () => {
+        store.dispatch(getMovies()).then(() => {
+            expect(store.getActions()).toEqual([ expectedAction ]);
+        })
+    })
+
+    it('delete movie', () => {

```

Comment:
if u need - it's possible  to write async tests as  `it('delete movie', async () => { `